data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -137,19 +137,59 @@ t_result_with_preprint_url_and_has_evaluations AS (
       WHEN latest_biorxiv_medrxiv_version.preprint_url IS NOT NULL THEN 'latest_biorxiv_medrxiv_version'
     END AS preprint_url_source,
 
-    (ARRAY_LENGTH(result.evaluations) > 0) AS has_evaluations
+    (ARRAY_LENGTH(result.evaluations) > 0) AS has_evaluations,
 
   FROM t_result_with_sorted_evaluations AS result
   LEFT JOIN t_latest_biorxiv_medrxiv_api_response_version_by_doi AS latest_biorxiv_medrxiv_version
     ON latest_biorxiv_medrxiv_version.doi = result.preprint_doi
 ),
 
+t_result_with_preprints AS (
+  SELECT 
+    result.* EXCEPT(
+      preprint_url,
+      ejp_validated_preprint_url,
+      preprint_url_source,
+      preprint_doi,
+      preprint_doi_source,
+      preprint_doi_url
+    ),
+
+    CASE WHEN result.preprint_url_source = 'evaluations' 
```

Comment:
very minor: There must a way to do the same with less SQL. I am just not sure now how. But maybe something to look into in the future.